src/ResourceManager/Compute/Stack/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1

```diff
@@ -143,17 +140,17 @@ function Create-VirtualMachine($rgname, $vmname, $loc)
 
     # Storage Account (SA)
     $stoname = 'sto' + $rgname;
-    $stotype = 'Standard_GRS';
-    $sa = New-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype;
+    $stotype = 'Standard_LRS';
+    New-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype;
     Retry-IfException { $global:stoaccount = Get-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname; }
     $stokey = (Get-AzureRmStorageAccountKey -ResourceGroupName $rgname -Name $stoname).Key1;
 
     $osDiskName = 'osDisk';
     $osDiskCaching = 'ReadWrite';
-    $osDiskVhdUri = "https://$stoname.blob.core.windows.net/test/os.vhd";
-    $dataDiskVhdUri1 = "https://$stoname.blob.core.windows.net/test/data1.vhd";
-    $dataDiskVhdUri2 = "https://$stoname.blob.core.windows.net/test/data2.vhd";
-    $dataDiskVhdUri3 = "https://$stoname.blob.core.windows.net/test/data3.vhd";
+    $osDiskVhdUri = "https://$stoname.blob.$env:STORAGEENDPOINTSUFFIX/test/os.vhd";
```

Comment:
@deepaft any reason to use an environment variable for this value?

Reply:
All environment specific values are passed in the tests through environment variables, so passed this too through environment variable. Each azure stack environment has a different storage endpoint suffix.

Reply:
@deepaft where is this environment variable set? We need to make sure it's easy for anyone to come in and re-record this test(s) if they need to